src/graph/backend/dnnl/kernels/kernel_base.cpp

```diff
@@ -43,6 +43,24 @@ bool kernel_base_t::enabled_constant_cache() const {
     return enabled;
 }
 
+size_t kernel_base_t::encode_constant_cache_key(
+        const std::vector<tensor_t> &inputs, size_t cache_key) const {
+    // Encode the constant memory address into cache key for differentiation
+    size_t encoded_cache_key = cache_key;
+    for (const auto &in : inputs) {
+        if (in.get_logical_tensor().property
+                == dnnl_graph_tensor_property_t::
+                        dnnl_graph_tensor_property_constant) {
```

Comment:
Firstly, it's suggested to use internal types. `property_type::constant` for this case. Secondly, please consider using the shortcut of `logical_tensor_wrapper::is_constant()`: https://github.com/oneapi-src/oneDNN/blob/22fdb3aeef4d42fe17de7d3e0152637d1498b3b6/src/graph/interface/logical_tensor.hpp#L84

Reply:
Changed, thanks!